src/Manager.java

```diff
@@ -8,35 +8,57 @@
     private String name;
     private Hotel hotel;
 
-    void Manager(String name, Hotel hotel) {
+    void Manager(String name, Hotel hotel)
+    {
         this.name = name;
         this.hotel = hotel;
     }
 
-    Manager(String name) {
+    Manager(String name)
+    {
         this.name = name;
     }
 
-    void Manager(Hotel hotel) {
+    void Manager(Hotel hotel)
+    {
         this.hotel = hotel;
     }
 
-    String getName(){
+    String getName()
+    {
         return this.name;
     }
 
-    String getHotel(){
+    String getHotel()
+    {
         return this.hotel.getName();
     }
 
-    ArrayList<Room> getHotelRooms(){
+    ArrayList<Room> getHotelRooms()
+    {
         return this.hotel.getRooms();
     }
 
-    Room findPerfectFitRoomOrReturnNull(String from, String to, int numberOfCouples, int numberOfPeopleSleepingAlone) throws ParseException {
-        Date dateFrom = new SimpleDateFormat("yyyy/mm/dd").parse(from);
-        Date dateTo = new SimpleDateFormat("yyyy/mm/dd").parse(to);
-        return this.hotel.findPerfectFitRoomOrReturnNull( dateFrom, dateTo, numberOfCouples, numberOfPeopleSleepingAlone);
+    Date stringToDate(String stringDate)
```

Comment:
Same method as in Room, I think. Try to make it a common class or method, which can be static and use that instead of duplicating the code to all classes in which you need it.